src/clause.h

```diff
@@ -256,10 +258,14 @@ class Clause
         _used_in_xor = false;
         _gauss_temp_cl = false;
         reloced = false;
+        isAtmost = atmost;
 
         for (uint32_t i = 0; i < ps.size(); i++) {
             getData()[i] = ps[i];
         }
+
+        if (is_atmost())
+            getData()[mySize] = Lit::toLit(0);
```

Comment:
It's a _really_ bad idea to skip on adding a single 32b value to all clauses to support atMost. Your speed penalty will be something like 0.1%, it's essentially noise. But you now have to have 1000 hacks and everything breaks all the time. I don't understand why you are doing this to yourself :)